openedx_events/learning/data.py

```diff
@@ -146,32 +146,37 @@ class DiscussionTopicContext:
     Context for linking the external id for a discussion topic to its associated usage key.
 
     Arguments:
-        title (str): is cached to improve the performance.
-        usage_key (str): usage key.
-        group_id(int): can be used for providers that don't internally support cohorting.
-        external_id(str): store the commentable id.
+        title (str): title of the discussion. This field is cached to improve the performance, since otherwise we'd
+        need to look it up in the course structure each time.
+        usage_key (str): unit location.
+        group_id (int): can be used for providers that don't internally support
+        cohorting but we can emulate that with different contexts for different groups.
+        external_id (str): store the commentable id that is used by cs_comments_service.
+        ordering (int): represent the position of the discussion topic.
     """
 
     title = attr.ib(type=str)
     usage_key = attr.ib(type=UsageKey, default=None)
     group_id = attr.ib(type=int, default=None)
     external_id = attr.ib(type=str, default=None)
+    ordering = attr.ib(type=int, default=None)
```

Comment:
Thanks @mariajgrimaldi, I see the missing part, for that reason generate the test mistakes in edx-platform